src/plugins/image/image-editor.js

```diff
@@ -183,7 +183,7 @@ function ImageEditor({ content, onContentChanged }) {
 
   const handleOrientationValueChanged = event => {
     const { value } = event.target;
-    const newEffect = { effect, orientation: value };
+    const newEffect = { ...effect, orientation: value };
```

Comment:
Five month old bug, mind you...

Reply:
Phew :open_mouth: At least now you can't save that anymore due to validation.

Reply:
True, good point!